test/reductions/vass/reproducer-tuple-with-array.skipif

```diff
@@ -0,0 +1,2 @@
+# Do not bother under non-valgrind configurations
+CHPL_TEST_VGRND_EXE != on
```

Comment:
Why not run it in all configurations? It seems like a simple enough test, and we expect it to pass whether valgrind or no, right?